Common/models/user.model.ts

```diff
@@ -0,0 +1,11 @@
+import {BaseEntity, Column, Entity, PrimaryColumn, Unique} from "typeorm";
+@Entity({ name: "user_model" })
+@Unique(["id"])
+export default class UserModel extends BaseEntity {
```

Comment:
We need a way to tie this to an actual login. Maybe we use discord id? Also we can try to add a auth override here. So if we need a certain user to have additional privs we can add it. So two more columns, a discord id and an authLevel which will be an optional number.

Reply:
Yea this model is WIP, ill actually implement it when i do userfacing work